servant-pt1.cabal

```diff
@@ -38,20 +38,32 @@ library
                      , say
                      , servant >= 0.15
                      , servant-server
+                     , stm
                      , string-conversions >= 0.4.0.1
                      , text >= 1.2.3.1
                      , text-show
                      , time >= 1.8.0.2
                      , wai
                      , warp
   hs-source-dirs:      src
+  exposed-modules:     Database
```

Comment:
In general, you want to have a library with most of your modules and most of the code.  Then, you want to import that from your executable below (which should live in a different directory).

Reply:
Oh, also, sometimes it is nice to create a small module namespace for your modules, so instead of just calling them `Database`, `Server`, etc, call them something like `MyProj.Database`, `MyProj.Server`, etc.